models/platform-limits/src/business-units/transformers.ts

```diff
@@ -0,0 +1,46 @@
+import { Transformer } from '@commercetools-test-data/core';
+import {
+  TBusinessUnitLimitsProjection,
+  TBusinessUnitLimitsProjectionGraphql,
+} from './types';
+
+const transformers = {
+  default: Transformer<
+    TBusinessUnitLimitsProjection,
+    TBusinessUnitLimitsProjection
+  >('default', {
+    buildFields: [
+      'maxDivisions',
+      'maxDepthLimit',
+      'maxAssociates',
+      'maxAssociateRoles',
```

Comment:
**nit**: we could extract this list to a constant in the file and reuse it in the three transformers.

Reply:
done https://github.com/commercetools/test-data/pull/655/commits/d01351de3bd1ac841ec8139721c78efb1fbcad99